pyproject.toml

```diff
@@ -32,17 +32,6 @@ classifiers = [
 ]
 
 [project.optional-dependencies]
```

Comment:
Shouldn't this also be gone?

Reply:
not sure about "should" but I will remove it